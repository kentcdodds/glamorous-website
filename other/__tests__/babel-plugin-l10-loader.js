import pluginTester from 'babel-plugin-tester'
import plugin from '../babel-plugin-l10n-loader'

pluginTester({
  plugin,
  snapshot: true,
  tests: [{fixture: require.resolve('./fixtures/some-component')}],
})
