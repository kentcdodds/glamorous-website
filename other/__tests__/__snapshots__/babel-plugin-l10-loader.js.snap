// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. babel-plugin-l10n-loader 1`] = `
"
/* eslint no-unused-vars:0 */
import fs from 'fs'
import content1 from './content'
import content2 from './content/index.md'

const path = require('path') // no change here

const content3 = require('./content')
const content4 = require('./content/index.md')

const array = [require('./content'), require('./content/index.md')]

function someFn() {
  const content5 = require('./content')
  const content6 = require('./content/index.md')
  const array2 = [require('./content'), require('./content/index.md')]
}

      ↓ ↓ ↓ ↓ ↓ ↓

/* eslint no-unused-vars:0 */
import fs from 'fs';
const content1 = { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content.js\\" }, \\"title\\": \\"English title\\" };
const content2 = { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content/index.md\\" }, \\"markdown\\": \\"<!-- removed yaml -->\\\\n\\\\n# english markdown\\\\n\\", \\"title\\": \\"English title\\", \\"contributors\\": [\\"someone\\", \\"someoneelse\\"] };


const path = require('path'); // no change here

const content3 = { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content.js\\" }, \\"title\\": \\"English title\\" };
const content4 = { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content/index.md\\" }, \\"markdown\\": \\"<!-- removed yaml -->\\\\n\\\\n# english markdown\\\\n\\", \\"title\\": \\"English title\\", \\"contributors\\": [\\"someone\\", \\"someoneelse\\"] };

const array = [{ \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content.js\\" }, \\"title\\": \\"English title\\" }, { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content/index.md\\" }, \\"markdown\\": \\"<!-- removed yaml -->\\\\n\\\\n# english markdown\\\\n\\", \\"title\\": \\"English title\\", \\"contributors\\": [\\"someone\\", \\"someoneelse\\"] }];

function someFn() {
  const content5 = { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content.js\\" }, \\"title\\": \\"English title\\" };
  const content6 = { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content/index.md\\" }, \\"markdown\\": \\"<!-- removed yaml -->\\\\n\\\\n# english markdown\\\\n\\", \\"title\\": \\"English title\\", \\"contributors\\": [\\"someone\\", \\"someoneelse\\"] };
  const array2 = [{ \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content.js\\" }, \\"title\\": \\"English title\\" }, { \\"meta\\": { \\"filename\\": \\"/other/__tests__/fixtures/content/index.md\\" }, \\"markdown\\": \\"<!-- removed yaml -->\\\\n\\\\n# english markdown\\\\n\\", \\"title\\": \\"English title\\", \\"contributors\\": [\\"someone\\", \\"someoneelse\\"] }];
}
"
`;
